include/SyntaxTree.h

```diff
@@ -40,16 +40,16 @@ class SyntaxTree {
     }
 
     /// Creates a syntax tree by guessing at what might be in the given source snippet.
-    static SyntaxTree fromText(StringRef text) {
-        return fromText(text, getDefaultSourceManager());
+    static SyntaxTree fromText(StringRef text, bool throwDiagnostics = false) {
```

Comment:
I don't think this should be added to this API... the caller can easily check if diagnostics().empty() is false and choose to throw if it wants.